### PR TITLE
feat: mll logo as default table loading indicator

### PR DESCRIPTION
**Pull request by mic-web**

### CODE REVIEW

File: src/Spinner/MllSpinnerIcon.tsx [hunk 1]
```diff
@@ -19,14 +19,17 @@ const SpinningSvg = styled.svg`
...
   </SpinningSvg>
 );
 
+const MllSpinnerIconComponent: IconComponentProps['component'] = () =>
```
Comment (spawnia) [inline src/Spinner/MllSpinnerIcon.tsx:28]: Can be inlined

File: src/Table/index.tsx [hunk 2]
```diff
@@ -22,8 +24,23 @@ export const StyledTable = styled(AntdTable)`
   props: TableProps<RecordType>,
 ) => ReactElement;
 
+function loadingProps<T>({ loading }: TableProps<T>): TableProps<T>['loading'] {
```
Comment (spawnia) [inline src/Table/index.tsx:27]: Less typing required when inlined, reuse is unlikely

File: src/Table/index.tsx [hunk 3]
```diff
@@ -22,8 +24,23 @@ export const StyledTable = styled(AntdTable)`
...
+  if (typeof loading === 'object') {
+    return {
+      spinning: loading.spinning,
+      indicator: loading.indicator ?? MllSpinnerSvg,
+      size: loading.size ?? 'large',
+    };
+  }
+  return {
```
Comment (spawnia) [inline src/Table/index.tsx:35]:
```suggestion
  }

  return {
```

**Review by spawnia — CHANGES_REQUESTED** [review]

File: src/Table/index.tsx [hunk 4]
```diff
@@ -25,5 +27,24 @@ export const StyledTable = styled(AntdTable)`
 export function Table<
   RecordType extends Record<string, unknown> = Record<string, unknown>,
 >(props: TableProps<RecordType>) {
-  return <StyledTable rowKey="id" {...props} />;
```
Comment (SimBig) [inline src/Table/index.tsx:28]: @mic-web FYI: another possibility is to wrap the styled tabel in a Spin component like I I did it for the `microplate`-component:
`
    <Spin
      spinning={props.loading ?? false}
      indicator={
        <MllSpinnerIcon
          style={{
            width: '2em',
          }}
        />
      }
    >
    `

**Review by spawnia — APPROVED** [review]

**Pull request merged**